blip-tap.md

```diff
@@ -1091,72 +1194,106 @@ receiving node:
 - MUST reject the payment if the `tap_rfq_id` has expired based on the posted
   `expiry` value.
 
-- MUST reject the payment the `amt_to_forward != (amt_asset_incoming * tick *
-  msat_multiplier) / accepted_rate_tick
-
-where:
-
-  * `amt_asset_incoming` is the HTLC value of the `asset_id` quote accepted in
-    the earlier RFQ round
-
-  * `msat_multiplier` is a factor used to scale from `BTC/asset` to
-    `msat/asset_id`, this value is constant within the protocol and is derived
-    by multiplying the number of sats per Bitcoin, by the amount of sats in an
-    msat: `msat_multiplier = 100_000_000 * 1000 = 100_000_000_000` (100
-    billion)
-
-    * As an example: 
-      * If the incoming channel is a USD backed asset, and wishes to send $1000
-        outbound, with a rate of $61234.95 (`accepted_rate_tick = 61234.95 *
-        tick = 612_349_500` then the forwarding node expects to be instructed
-        to send `1,633,054,326 mSAT` over the outgoing link:
-          * `amt_to_forward = ($1000 * tick * msat_multiplier) // 612_349_500`
-          * `amt_to_forward = (1_000 * 10_000 * 100_000_000_000) // 612_349_500`
-          * `amt_to_forward = (1000000000000000000) // 612_349_500`
-          * `amt_to_forward = 1_633_054_326`
+- MUST reject the payment if
+  ```
+  amt_msat_to_forward > (amt_asset_incoming / incoming_asset_to_btc) * btc_to_msat_multiplier
+  ```
+  This equation stipulates an upper bound on the HTLC msat value that the
+  liquidity provider should forward to the next hop.
+
+  In this equation, the right-hand side converts the incoming TAP asset amount
+  to BTC by dividing it by the asset-to-BTC conversion rate
+  (`incoming_asset_to_btc`). The resulting BTC value is then converted to
+  milli-satoshis (msats).
+
+  The terms in this equation are defined as follows:
+ 
+  * `amt_msat_to_forward` is the HTLC msat value that the liquidity provider
+    should forward to the next hop.
+
+  * `amt_asset_incoming` is the HTLC value of the incoming asset amount.
+
+  * `btc_to_msat_multiplier` is a constant used to convert BTC to
+    milli-satoshis (msats). It is defined as:
+    ```
+    btc_to_msat_multiplier = 100_000_000 * 1000 = 100_000_000_000`
+    ```
+
+    This constant combines two conversions:
+    1. BTC to Satoshis: 1 BTC = 100,000,000 satoshis.
+    2. Satoshis to msats: 1 satoshi = 1,000 msats.
+
+  * `incoming_asset_to_btc` represents the fractional conversion rate between
```

Comment:
nit: should we use the shorter `in_asset_to_btc` term we've used in the messages above?

Reply:
I used "incoming" here deliberately. This section covers what the receiving node (i.e. the edge node) must do. So I think this is actually the out-asset as found in the RFQ messages. I think it's more confusing to use out-asset here. What do you think we should use here?

Reply:
Ok, makes sense.